lte/gateway/c/core/oai/tasks/amf/amf_app_handler.cpp

```diff
@@ -1215,9 +1215,8 @@ static int paging_t3513_handler(zloop_t* loop, int timer_id, void* arg) {
         LOG_AMF_APP,
         "T3513: Maximum retires done hence Abort the Paging Request "
         "procedure\n");
-    OAILOG_FUNC_RETURN(LOG_NAS_AMF, RETURNok);
   }
-  OAILOG_FUNC_RETURN(LOG_NAS_AMF, RETURNok);
+  OAILOG_FUNC_RETURN(LOG_NAS_AMF, rc);
```

Comment:
rc is initialized to RETURNerror, if you do not enter the if, here your modification change the returned value of function, I did not dig into the consequence.

Reply:
This should always return okay otherwise zmq loop will exist treating this as the intent. I suggest that we remove rc completely and return Ok always.

Reply:
@ulaskozat 

I agree w/ your comment. My point I wanted to make w/ this PR is that we are accepting code in w/o paying attention to compilation warnings.

One of them was quite important --> `ue_context->mm_state == REGISTERED_CONNECTED;`

I was away for a few weeks and the number rises to `22`....

For me, this is not acceptable that every 2 months, I have to do such a PR to fix other's people work.

Raphael